Makefile

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+.PHONY: format
+format:
+	yapf -i --style='{based_on_style: google, indent_width: 2}' -r pipeline_dp tests
```

Comment:
can you add `yapf` to poetry dev too?

Reply:
👀 Missed that, thank you!

Reply:
👋 I ran `poetry add yapf` which added the line `yapf = "^0.31.0"`.
Please let me know if there's another step I should run. Thanks!